reports/RJ722-II.md

```diff
@@ -0,0 +1,439 @@
+---
+layout: report
+permalink: /GSoC/2018/StatusReport/RahulJha
+student: Rahul Jha
+organisation: coala
+organisation_link : https://coala.io
+project: Improve Whitelisting Infrastructure for Vulture
+project_link: https://summerofcode.withgoogle.com/projects/#5620743946108928
+report_year: 2018
+tarball: https://drive.google.com/file/d/12HgBb2Wdjl6Zzm7mDqCt13ke8VHj7_Bb/view
+mentors: >
+ [Jendrik Seipp](https://github.com/jendrikseipp) & [Abdeali Kothari](https://github.com/AbdealiJK) & [Florian Bruhin](https://github.com/The-Compiler)
+phase:
+ - Bonding : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/28
+ - Phase 1 : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/29
+ - Phase 2 : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/30
+ - Phase 3 : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/31
+bio: >
+ Rahul is currently a junior, studying Electronics Engineering at Zakir Hussain
+ College of Engineering and Technology, Aligarh Muslim University. He loves the
+ ideology behind Free and Open Source Software and reads about the history of
+ this revolutionary movement in his leisure time.
+social:
+ - GitHub:
+   - username: RJ722
+   - link: https://github.com/RJ722
+ - GitLab:
+   - username: RJ722
+   - link: https://gitlab.com/RJ722
+ - Gitter:
+   - username: RJ722
+   - link: https://gitter.im/RJ722
+ - Openhub:
+   - username: RJ722
+   - link: https://openhub.net/accounts/RJ722
+email: rj722@protonmail.com
+blog: https://rj722.github.io/blog
+activity:
+ - 1:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/b8a8ec8b42b24287560fd07ff3cafe30e92fb585
+   - details: Initial commit
+
+ - 2:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/43f6dae0818cdba691fb3f78e6bb4381867beeb3
```

Comment:
doesnt count.

this should have been automated with cookiecutter or moban

Reply:
> this should have been automated with cookiecutter or moban

Ahh, I wasn't aware of this. I wrote a script which looked for every commit with the author "RJ722" or "Rahul Jha" and spitted this. Not very smart, sorry - I'd be removing these.